python/test/unit/language/test_pipeliner.py

```diff
@@ -428,8 +428,11 @@ def indirect_matmul_kernel(
 @pytest.mark.parametrize("BLOCK_M, BLOCK_N, BLOCK_K", [(128, 128, 128), (128, 128, 64), (128, 64, 128)])
 @pytest.mark.parametrize("num_stages", [1, 3, 5])
 def test_indirect_matmul(BLOCK_M, BLOCK_N, BLOCK_K, num_stages, device):
-    if num_stages > 3 and is_hip():
-        pytest.skip("Not enough shared memory on HIP.")
+    if is_hip():
+        if num_stages > 3:
+            pytest.skip("Not enough shared memory on HIP.")
+        if num_stages == 3 and BLOCK_M == BLOCK_N == BLOCK_K == 128:
+            pytest.skip("Not enough shared memory on HIP.")
```

Comment:
This is a [new fail](https://github.com/triton-lang/triton/actions/runs/14272019120/job/40007197632#step:12:532) that looks like 

```
FAILED language/test_pipeliner.py::test_indirect_matmul[3-128-128-128] - triton.runtime.errors.OutOfResources: out of resource: shared memory, Required: 66048, Hardware limit: 65536. Reducing block sizes or `num_stages` may help.
```

I believe this is because that IR now successfully generates buffer ops where prior it did not. Will double check.

Reply:
fixed by @peterbell10 in https://github.com/triton-lang/triton/pull/6393